README.md

```diff
@@ -256,7 +256,7 @@ $ nixpkgs-review pr -p nixosTests.ferm 47077
 
 ## Ignoring ofborg evaluations
 
-By default, nixpkgs-review will use ofborg's evaluation result if available to
+By default nixpkgs-review will use ofborg's evaluation result if available to
```

Comment:
I think the prepositional phrase warrants a comma. Because the rest of the sentence makes sense without, he just wanted to stress that this can change.